cterasdk/edge/shares.py

```diff
@@ -74,6 +76,7 @@ def add(self,
         param.comment = comment
         Shares._validate_acl(acl)
         param.acl = [acl_entry.to_server_object() for acl_entry in acl]
+        param.trustedNFSClients = [client.to_server_object() for client in (trusted_nfs_clients or [])]
```

Comment:
Is it OK to send an empty array. In other places we added a field only if needed

Reply:
Yes. It's idd but the schema requires an empty list